ssl/record/ssl3_record.c

```diff
@@ -272,9 +272,10 @@ int ssl3_get_record(SSL *s)
                 thisrr->rec_version = version;
 
                 /*
-                 * Lets check version. In TLSv1.3 we ignore this field. For an
-                 * HRR we haven't actually selected TLSv1.3 yet, but we still
-                 * treat it as TLSv1.3, so we must check for that explicitly
+                 * Lets check version. In TLSv1.3 we ignore this field. For the
+                 * ServerHello after an HRR we haven't actually selected TLSv1.3
+                 * yet, but we stilltreat it as TLSv1.3, so we must check for
```

Comment:
You might add the space in "still treat" before merging.

Reply:
Fixed